src/fragments/gen2/quickstart/create-nuxtjs-project.mdx

```diff
@@ -0,0 +1,14 @@
+## Create a project
```

Comment:
remove this fragment and move it directly into the page.

Reply:
especially avoid fragments if the content is only used once.

Reply:
Done!